core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -2742,6 +2742,11 @@ private[spark] object Utils extends Logging {
     }
   }
 
+  def getTimeZone: TimeZone = {
+    val sparkConf = new SparkConf(false).loadFromSystemProperties(true)
```

Comment:
Can we make SparkConf as a input param instead of create a new instance for every function call?